config/config.go

```diff
@@ -460,7 +460,10 @@ type Config struct {
 	DisableKeepAlives            bool `yaml:"disable_keep_alives"`
 	MaxIdleConns                 int  `yaml:"max_idle_conns,omitempty"`
 	MaxIdleConnsPerHost          int  `yaml:"max_idle_conns_per_host,omitempty"`
-	MaxHeaderBytes               int  `yaml:"max_header_bytes"`
+	MaxRequestHeaderBytes        int  `yaml:"max_header_bytes"`
+	MaxResponseHeaderBytes       int  `yaml:"max_response_header_bytes"`
+	MaxRequestHeaders            int  `yaml:"max_request_headers"`
```

Comment:
for consistency's sake, we could call this `max_headers`?

I know the missing `request` in `max_(request_)header_bytes` is for backward compatibility, but having `request` in one property and not the other seems also inconsistent.

Reply:
See also my [previous comment](https://github.com/cloudfoundry/gorouter/pull/453#issuecomment-2467674019). If we are fine with adjusting the coding and only guaranteeing the compatibility on release level, that would be my preferred solution.

Reply:
This might also be a question for the routing-release, right?

In the end we don't intend to configure gorouter directly but only via the routing release. The conversion / could also be done there, i.e. legacy parsing of `max_header_bytes`, but allow `max_request_header_bytes` also, and map both to gorouter's `max_request_header_bytes`?

Reply:
That's probably the best solution, I'll see what I can do.

Reply:
See the routing-release PR: https://github.com/cloudfoundry/routing-release/pull/443